src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethodIgnoreThrows.java

```diff
@@ -0,0 +1,79 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.function.Function;
+
+public class InputJavadocMethodIgnoreThrows {
```

Comment:
please add above comment and place there config as we do in others recent Inputs.

please also add to our Inputs a bit more test cases,  3 test cases is not enough for such controversial issue.
Please take bunch of interesting cases from your regression report. We should not rely on regression testing that much, we need to grow out Inputs code base.

Reply:
I tried to add more tests but some of the results were not what I was expecting, opened issue #8117 for it.

I'll hold on to this change until #8117 is confirmed or resolved.

Reply:
issue is approved.

Reply:
Unfortunately I can't wait for #8117 to be fixed first because it also causes some false positives within Checkstyle itself as well, using anonymous classes. So fix for this issue needs to be merged first. https://github.com/checkstyle/checkstyle/blob/2e8c277c181487d81231170ff4b959edc008114e/src/main/java/com/puppycrawl/tools/checkstyle/JavaParser.java#L91-L96

I've updated this PR to exclude anonymous and local classes as well (previously covered only lambdas), regression will be up soon.

It's a bit hard to come up with more complex input cases as violations keep getting ignored due to the problem in #8117. I'll do what I can, meanwhile fixes for the other points have been pushed and ready for review.

Reply:
JavadocMethod is kind deprecated Check, that waits time to be reimplemented to AST based validation, because current implementation is prone to false-positives and complications in analysis. But UTs and inputs will be reused.

Reply:
ok, but each Input still need config copy in comment over class.
One day in future we will have UT that takes Input file , take config from it, run, and check that violations are on the same lines. So maintainers primary role will be to simply review Inputs.